create-image.sh

```diff
@@ -63,12 +63,18 @@ case "$JETSON_NANO_BOARD" in
         printf "OK\n"
         ;;
 
-    jetson-nano)
-        printf "Create image for Jetson nano board"
+    jetson-nano-4gb-a02)
+        printf "Create image for Jetson nano 4GB (A02) board"
         ./jetson-disk-image-creator.sh -o jetson.img -b jetson-nano -r 200
         printf "OK\n"
         ;;
 
+    jetson-nano-4gb-b01)
+        printf "Create image for Jetson nano 4GB (B01) board"
+        ./jetson-disk-image-creator.sh -o jetson.img -b jetson-nano -r 300
+        printf "OK\n"
+        ;;
+
```

Comment:
Hi @ksachdeva 

Thanks for your contributions.

I think it's great idea to support the new Jetson boards. However, I would suggest some changes so we support all the different revisions of the boards. 

I would go with defining a new optional env variable `JETSON_NANO_REVISION` and this variable could take `200` or `300` values or any valid values for the revision (should take a look at the nvidia doc to see what are the valid values). I would suggest to have `200` for the default as I assume this revision is more common. And then the script will be:
```
./jetson-disk-image-creator.sh -o jetson.img -b jetson-nano -r  $JETSON_NANO_REVISION
```
Once those changes are done, then I can merge it and update my blog post

Reply:
I have pushed the requested changes.  I have kept the default to 200 as per your recommendation however my understanding is that the board rev 300 is now more common and it is the only one Nvidia makes but I have not done extensive research on it.